service/controller/resource/app/create.go

```diff
@@ -158,6 +158,17 @@ func (r *Resource) newApp(ctx context.Context, cr capi.Cluster, appSpec AppSpec)
 		kubeConfig = v1alpha1.AppSpecKubeConfig{
 			InCluster: true,
 		}
+	} else if key.IsBundle(appSpec.App) {
+		kubeConfig = v1alpha1.AppSpecKubeConfig{
+			InCluster: true,
+			Context: v1alpha1.AppSpecKubeConfigContext{
```

Comment:
Do we need  this information, as well as the secret name, when we have in-cluster app? The bundle App CR is installed in the Management Cluster context.

Reply:
Maybe not but this is how it's done for the security packs so I did the same. Let me correct it

Reply:
Fixed in latest commig